_integration-test/run.sh

```diff
@@ -51,7 +53,7 @@ echo "${_endgroup}"
 
 echo "${_group}Running tests ..."
 get_csrf_token() { awk '$6 == "sc" { print $7 }' $COOKIE_FILE; }
-sentry_api_request() { curl -s -H 'Accept: application/json; charset=utf-8' -H "Referer: $SENTRY_TEST_HOST" -H 'Content-Type: application/json' -H "X-CSRFToken: $(get_csrf_token)" -b "$COOKIE_FILE" -c "$COOKIE_FILE" "$SENTRY_TEST_HOST/api/0/$1" ${@:2}; }
+sentry_api_request() { curl -s -H 'Accept: application/json; charset=utf-8' -H "Referer: $SENTRY_TEST_HOST" -H 'Content-Type: application/json' -H "X-CSRFToken: $(get_csrf_token)" -b "$COOKIE_FILE" -c "$COOKIE_FILE" "$SENTRY_TEST_HOST/$1" ${@:2}; }
```

Comment:
We make a request against relay so we must make this generic over the API path used (see where we calculate `EVENT_ID` below)